pkg/project/project.go

```diff
@@ -5,7 +5,7 @@ var (
 	gitSHA      = "n/a"
 	name        = "app-operator"
 	source      = "https://github.com/giantswarm/app-operator"
-	version     = "n/a"
+	version     = "1.0.0"
```

Comment:
adding `1.0.0` and compare it with version labels in CR.

Reply:
This will only work for the first release. As soon as we need to create `1.0.1` it doesn't match any more.

I added a ChartVersion func to chart-operator. I think we need AppVersion here.

https://github.com/giantswarm/chart-operator/blob/ae3cb2f4c26821f66aa47199bdbc64855580d402/pkg/project/project.go#L11-L16

Reply:
Got it, leave it as it 1.0.0 but would introduce AppVersion in this file.

Reply:
App version and chart version are equal for production releases.

Reply:
And one more thing. The orb will fail if value of `version` in `project.go` and the value in `appVersion` in `Chart.yaml` are not equal.

Reply:
Example https://circleci.com/gh/giantswarm/release-operator/2311